dspace/janitor.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "DSpace",
+  "description": "DSpace is a turnkey institutional repository application.",
+  "icon": "https://janitor.technology/img/dspace.svg",
+  "docker": {
+    "image": "janitortechnology/dspace"
+  },
+  "ports": {
+    "22": {
+      "label": "SSH",
+      "proxy": "none"
+    },
+    "5432": {
+      "label": "DB",
+      "proxy": "https"
+    },
+    "8080": {
```

Comment:
Nit: I think the VNC server actually runs on port `8088` (and I suspect that you were trying to add port `8080` in addition to the VNC port `8088`, if so, please create two separate entries here).